package.json

```diff
@@ -27,77 +27,66 @@
     "functional-programming"
   ],
   "scripts": {
-    "example": "ts-node --project tsconfig.examples.json",
-    "version": "changeset version && pnpm install --no-frozen-lockfile && pnpm run docs-update",
-    "release": "pnpm run build && changeset publish",
-    "clean": "rimraf build tsbuildinfo dist .cache",
-    "build": "pnpm build-all && pnpm build-pack",
-    "build-cjs": "babel build/esm --config-file ./.babel.cjs.json --out-dir build/cjs --out-file-extension .js --source-maps",
-    "build-mjs": "babel build/esm --config-file ./.babel.mjs.json --out-dir build/mjs --out-file-extension .mjs --source-maps",
-    "build-post": "build-utils pack-v4",
-    "build-pack": "concurrently \"pnpm build-cjs\" \"pnpm build-mjs\" && pnpm build-post",
-    "build-all": "tsc -b tsconfig.json",
-    "build-watch": "tsc -b tsconfig.json --watch",
-    "lint": "eslint src/**/* test/**/* examples/**/*",
-    "autofix": "pnpm lint --fix",
-    "tc": "tsc --noEmit",
-    "docs": "docgen",
-    "docs-update": "git add --force --all docs/modules || true",
-    "circular": "madge --ts-config ./tsconfig.madge.json --circular --no-color --no-spinner --warning build/esm",
+    "clean": "node scripts/clean.mjs",
+    "codegen": "pnpm --recursive --parallel run codegen",
+    "build": "tsc -b tsconfig.build.json",
+    "circular": "node scripts/circular.mjs",
     "test": "vitest",
-    "coverage": "vitest run --coverage"
+    "coverage": "vitest --coverage",
+    "check": "tsc -b tsconfig.json",
+    "lint": "eslint \"**/{src,test,examples,scripts,dtslint}/**/*.{ts,mjs}\"",
+    "lint-fix": "pnpm lint --fix",
+    "docgen": "docgen",
+    "dtslint": "pnpm --recursive --parallel run dtslint",
+    "dtslint-clean": "dtslint --installAll",
+    "changeset-version": "changeset version && node scripts/version.mjs",
+    "changeset-publish": "pnpm codemod && pnpm lint-fix && pnpm build && TEST_DIST= pnpm vitest && changeset publish"
   },
-  "exports": {
-    ".": {
-      "require": "./build/cjs/index.js"
-    },
-    "./*": {
-      "require": "./build/cjs/*.js",
-      "import": "./src/*.ts"
-    }
-  },
-  "dependencies": {
-    "@effect/data": "~0.17.1",
-    "@effect/io": "~0.36.1"
+  "peerDependencies": {
+    "effect": "^3.10.19"
```

Comment:
We may want to update the peer dependence to the latest effect (i.e. 3.11).

Reply:
Of course 👍 , hard to keep up with you guys... ;)